floodviz/templates/index.html

```diff
@@ -3,12 +3,21 @@
 <head>
 	<meta charset="UTF-8">
 	<link type="text/css" href="{{ url_for('static', filename='bower_components/bootstrap/dist/css/bootstrap.css') }}" />
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/mapstyle.css') }}" />
```

Comment:
Looks like you are using spaces rather than tabs. Use tabs for all files except the python code.